python/lsst/meas/mosaic/chebyshev.cc

```diff
@@ -0,0 +1,53 @@
+/*
+ * LSST Data Management System
+ * Copyright 2008-2017  AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+#include "lsst/meas/mosaic/chebyshev.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace mosaic {
+
+PYBIND11_PLUGIN(chebyshev) {
+    py::module mod("chebyshev");
+
+    mod.def("T0", &T0);
+    mod.def("T1", &T1);
+    mod.def("T2", &T2);
+    mod.def("Tn", &Tn);
+
+    py::class_<Chev, std::shared_ptr<Chev>> cls(mod, "Chev");
+
+    cls.def(py::init<int>(), "order"_a);
+    cls.def_readwrite("order", &Chev::order);
+    cls.def_readwrite("coeffs", &Chev::coeffs);
+
+    return mod.ptr();
+}
```

Comment:
I don't think this header needs to be wrapped at all - I don't see it in the `%include`s in `mosaicLib.i`.